bigcommerce/resources/base.py

```diff
@@ -124,4 +124,11 @@ def delete(self):
     @classmethod
     def delete_all(cls, parentid, connection=None):
         path = "%s/%s/%s" % (cls.parent_resource, parentid, cls.resource_name)
-        return cls._make_request('DELETE', path, connection)
\ No newline at end of file
+        return cls._make_request('DELETE', path, connection)
+
+class CountableApiResource(ApiSubResource):
```

Comment:
That makes sense to me. The only resources with a count endpoint are `Customer`, `Product`, and `Order`?